### PR TITLE
feat(ssi credential): created new credential page

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Created new page and connected with business logic

## Why

This page is created to see all credential request and whether admin wants to approve/decline

## Issue

CPLP-2924

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: src/components/pages/AdminCredential/AdminCredentialElements.tsx [hunk 1]
```diff
@@ -76,51 +119,57 @@ export default function AdminCredentialElements() {
...
+      field: 'companyId',
       headerName: t('content.adminCertificate.table.companyInfo'),
       flex: 2,
     },
     {
-      field: 'applicationId',
+      field: 'useCase',
       headerName: t('content.adminCertificate.table.certificate'),
       flex: 1.5,
+      renderCell: ({ row }: { row: any }) => <>{row.useCase ?? 'N/A'}</>,
```
Comment (oyo) [inline src/components/pages/AdminCredential/AdminCredentialElements.tsx:136]: isn't row of type CredentialData here?

File: src/assets/locales/de/main.json [hunk 2]
```diff
@@ -1464,6 +1464,29 @@
...
+      "table": {
+        "crendentialType": "Type",
+        "companyInfo": "Company Info",
+        "certificate": "Certificate",
+        "document": "Document"
+      },
+      "approvedMessage": "Credential Request Accepted successfully.",
+      "declinedMessage": "Credential Request Declined successfully.",
+      "errorMessage": "Something went wrong!"
+    },
```
Comment (oyo) [inline src/assets/locales/de/main.json:1489]: translate these lines to German to reduce the duplication. If that is planned to be done at a later time, maybe put `de: ` in front of every value

**Review by oyo — CHANGES_REQUESTED** [review]

**Comment by evegufy** [comment]
@nidhigarg-bmw @oyo I created https://github.com/eclipse-tractusx/portal-frontend/pull/147 to address sonarcloud duplications
cc: @jjeroch

**Review by oyo — CHANGES_REQUESTED** [review]
Locales are now excluded from Sonar checks. Push one dummy commit to re-trigger.

Also the PR title doesn't match conventional commit standards. Please check PR naming scheme here: https://github.com/eclipse-tractusx/portal-frontend/pulls?q=is%3Apr
In doubt read more here: https://www.conventionalcommits.org/en/v1.0.0/

**Comment by oyo** [comment]
one file causing a merge conflict

**Review by oyo — APPROVED** [review]

**Pull request merged**